ansible/roles/common/tasks/main.yml

```
@@ -121,3 +121,7 @@
   authorized_key: key="{{ lookup('file', item) }}" user=deploy
   with_fileglob:
     - ssh_keys/*
+
+- name: enable systemctl services for deploy user
+  become: true
+  action: loginctl enable-linger deploy
```

```suggestion
  action: loginctl enable-linger deploy
  become: true
```
To ensure the action is run as root user.

---

Wait, the order matters?

---

Missed the actual `become: true`. Please ignore this.